app/assets/stylesheets/global.scss

```diff
@@ -84,8 +84,10 @@ button, .button {
   }
 }
 
+.spacer { @extend %shared-spacer; }
+
 .spacer-alt {
-  @extend .spacer;
+  @extend %spacer;
```

Comment:
Should this be `%shared-spacer` not `%spacer`?

Reply:
yes, which is probably why tests are crashing

Reply:
```suggestion
  @extend %shared-spacer;
```